readthedocs/oauth/services/__init__.py

```diff
@@ -10,4 +10,8 @@
     getattr(settings, 'OAUTH_BITBUCKET_SERVICE',
             'readthedocs.oauth.services.bitbucket.BitbucketService'))
 
-registry = [GitHubService, BitbucketService]
+GitLabService = import_by_path(
+    getattr(settings, 'OAUTH_GITLAB_SERVICE',
+            'readthedocs.oauth.services.gitlab.GitLabService'))
+
+registry = [GitHubService, BitbucketService, GitLabService]
```

Comment:
This is required, we use this pattern to override API actions on private resources on readthedocs.com.